astropy/time/core.py

```diff
@@ -119,14 +119,14 @@ def _represent_as_dict_attrs(self):
         return out + self._represent_as_dict_extra_attrs
 
     def __init__(self, bound=False):
-        super(MixinInfo, self).__init__(bound)
+        super().__init__(bound)
 
         # If bound to a data object instance then create the dict of attributes
         # which stores the info attribute values.
         if bound:
             # Specify how to serialize this object depending on context.
             # If ``True`` for a context, then use formatted ``value`` attribute
-            # (e.g. the ISO time string).  If ``False`` then use decimal jd1 and jd2.
+            # (e.g. the ISO time string).  If ``False`` then use jd1 and jd2.
```

Comment:
Out of scope, and I had written `decimal` for a reason to highlight that floating point numbers are being serialized in this case.  Maybe `float` would be a better word.

Reply:
OK, putting back float.